workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestForm/utils/submitForm.ts

```diff
@@ -71,6 +71,12 @@ export default async function submitForm(
 					);
 				}
 				else {
+					if (values.currencyExchangeRate !== 0) {
+						mdfRequestActivity.convertedMDFRequestedAmount =
+							mdfRequestActivity.mdfRequestAmount /
+							values.currencyExchangeRate;
+					}
+
```

Comment:
remove this logic from here, you can use DTO for that

Reply:
Done!